inductiva/resources/machines.py

```diff
@@ -155,6 +160,7 @@ def __init__(
         self.num_active_machines = min_machines
         self.__is_elastic = True
         self.spot = spot
+        self.num_machines = max_machines
```

Comment:
I think this more extra for the elastic class, no? Because, we are doing everything with min, max, active machines. Especially if we override the àctive_machines_to_str`.

Reply:
I agree, here the num_machines is unnecessary.